src/EditorFeatures/Core/IntelliSense/QuickInfo/IntellisenseQuickInfoBuilder.cs

```diff
@@ -109,8 +109,9 @@ internal static class IntellisenseQuickInfoBuilder
                 {
                     // We don't present additive-spans (like static/reassigned-variable) any differently, so strip them
                     // out of the classifications we get back.
+                    // Question - could we directly pass in quickInfoItem.RelatedSpans and take this out of above for loop?
```

Comment:
// Question - could we directly pass in quickInfoItem.RelatedSpans and take this out of above for loop?

Reply:
possibly.  though as this is quick info, it's likely very low cost here, so not essential to fix up.